CONTRIBUTING.md

```diff
@@ -140,16 +140,18 @@ Because SQL within JSON or YAML will not generally be syntax highlighted, indent
 
 ## Map Icon Guidelines
 
-Design Guidelines
-* SVG only
-* flat meaning a single colour, no gradients, no outlines
-* common canvas size, which is usually 14x14 px
-* single point of view and avoid use of perspective where possible
-
-Best Practices
-* align to pixel grid
-* clean design, so reduced complexity where possible
-
+* All new icons must be SVG format only.  The SVG must be saved as standards compliant SVG without any prioprietery tags.  In Inksapce software, you will need to Save As... Plain SVG format.
+* Icons must be use fills and not outlines.
```

Comment:
It needs to be clearer that this is a technical requirement to use SVG fills, not SVG strokes. Perhaps

> * Icons must use SVG fills only, not SVG strokes or any feature Mapnik does not support.

As a design, the bicycle icon has outlines, but they are implemented as SVG fills.